system/lib/compiler-rt/lib/asan/asan_thread.cc

```diff
@@ -297,10 +301,13 @@ void AsanThread::SetThreadStackAndTls(const InitOptions *options) {
   tls_end_ = tls_begin_ + tls_size;
   dtls_ = DTLS_Get();
 
+// XXX EMSCRIPTEN: No idea why this sometimes fails, so removing it.
+#if !SANITIZER_EMSCRIPTEN
```

Comment:
This sounds worth investigating further.

Reply:
Investigated and resulted in #8905. The hack has been removed.